src/runtime/virtcontainers/nydusd_other.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022 Apple Inc.
```

Comment:
happy new year?

Reply:
LOL, we talked about the same in https://github.com/kata-containers/kata-containers/pull/5984#discussion_r1062670516

Reply:
Need at least two weeks to adjust to there being a 3